python/ray/state.py

```diff
@@ -1120,6 +1185,19 @@ def node_ids():
     return node_ids
 
 
+def actors(actor_id=None):
+    """Fetch and parse the actor table info for one or more actor IDs.
```

Comment:
I realize that we do this in some of the other methods, but please don't use the term "actor table" in user-facing documentation. The idea of a "table" is confusing and really more of an implementation detail. The documentation should just use concepts that users think about like "actors".

Reply:
This applies in the args also